src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt

```diff
@@ -462,6 +466,17 @@ private fun getPowerDisplayPair() = listOf(
         ?: "§cOpen \"Your Bags\"!") to HorizontalAlignment.LEFT
 )
 
+private fun getTuningDisplayPair() = listOf(
```

Comment:
You could also maybe add the "Display numbers first" functionality, seems to be more inline with the rest of the display

Reply:
can you link me a line number where you use this?

Reply:
[here](https://github.com/hannibal002/SkyHanni/blob/70d28d6e2ec3fdb9babe632d49c36fad6e7f683c/src/main/java/at/hannibal2/skyhanni/features/gui/customscoreboard/ScoreboardElements.kt#L262) for example